### PR TITLE
Add extension to type imports for nodenext module resolution

### PR DESCRIPTION
**Pull request by kyryloonufriiev**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: packages/webdriverio/src/commands/element/nextElement.ts [hunk 1]
```diff
@@ -22,7 +24,7 @@
  * @type utility
  */
 
-export default function nextElement (this: WebdriverIO.Element) {
+export default function nextElement (this: WebdriverIO.Element): ChainablePromiseElement<any>  {
```
Comment (christian-bromann) [inline packages/webdriverio/src/commands/element/nextElement.ts:27]: Is this needed? TS figures this out automatically? Also we should avoid `any` if possible

File: packages/webdriverio/src/commands/element/parentElement.ts [hunk 2]
```diff
@@ -21,7 +23,7 @@
  * @return {Element}
  * @type utility
  */
-export default function parentElement (this: WebdriverIO.Element) {
+export default function parentElement (this: WebdriverIO.Element): ChainablePromiseElement<any>  {
```
Comment (christian-bromann) [inline packages/webdriverio/src/commands/element/parentElement.ts:26]: same

File: packages/webdriverio/src/commands/element/previousElement.ts [hunk 3]
```diff
@@ -21,7 +23,7 @@
  * @return {Element}
  * @type utility
  */
-export default function previousElement (this: WebdriverIO.Element) {
+export default function previousElement (this: WebdriverIO.Element): ChainablePromiseElement<any> {
```
Comment (christian-bromann) [inline packages/webdriverio/src/commands/element/previousElement.ts:26]: same

**Review by christian-bromann — COMMENTED** [review]
One comment

File: packages/webdriverio/src/commands/element/nextElement.ts (hunk 1 above)
Comment (kyryloonufriiev) [reply packages/webdriverio/src/commands/element/nextElement.ts:27]: When ts figuring it out it generates `import("../..").ChainablePromiseElement<WebdriverIO.Element>`, and this path causes problems in node16/nodenext module resolution.
I admit that I have to replace `any` with `WebdriverIO.Element`.

**Review by christian-bromann — APPROVED** [review]
👍  Thank you!

**Comment by kyryloonufriiev** [comment]
@christian-bromann When we can expect a new wdio version?

**Comment by christian-bromann** [comment]
Triggered a new release already!

**Pull request merged**